### PR TITLE
Prepare release v1.6.8

### PR DESCRIPTION
**Pull request by koic**

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## 1.6.8 / 2017-03-18
+
+* Changes and bug fixes since 1.6.7
+ * Better fix to support "Relation#count does not support finder options anymore in Rails [#787, #788]
+ * Run Oracle enhanced adapter unit tests using Travis CI [#789]
+ * Upgrade travis-oracle to Version 2.0.1 [#903]
+ * NoMethodError: undefined method `write' for nil:NilClass for serialized column [#798, #1004]
+ * Support connection strings in `DATABASE_URL1 [#1036]
```
Comment (yahonda) [inline History.md]: It was due to my typo in 1.7.5, let's fix for new entries. `DATABASE_URL`.

**Comment by yahonda** [comment]
I still do not know which entry should come first, oldest commit or important fixes for users.
In case bumping minor version such as 1.7.x to 1.8.0, Listed new features first. This time this is the bug fix version and not so many fixes included the current order should be fine.

**Comment by yahonda** [comment]
Once fixes to `DATABASE_URL`, we can go ahead to merge it.

File: History.md (hunk 1 above)
Comment (koic) [inline History.md]: Thanks for your review. I fixed it and git squashed.

**Comment by yahonda** [comment]
Thanks

**Comment by koic** [comment]
> I still do not know which entry should come first, oldest commit or important fixes for users.

Hmm, I don't care much about the order of bug fixes. However, I think that it is good to have a important (serious) bug fix at the top of the bug fix list. Because it is kind to the users.

> In case bumping minor version such as 1.7.x to 1.8.0, Listed new features first. 

Sure. I’ll keep it in mind.

> This time this is the bug fix version and not so many fixes included the current order should be fine.

I got it. Thanks for your review and merging.

**Pull request merged**